api/src/main/proto/stellarstation/api/v1/radio/radio.proto

```diff
@@ -161,6 +164,24 @@ message CCSDSTransferFrame {
   LineCoding line_coding = 6;
 }
 
+// https://gomspace.com/UserFiles/Subsystems/datasheet/gs-ds-nanocom-ax100-33.pdf
+//
+// ASM+Golay mode for GOM Space AX100 radio
+message ASMGolay {
+  // Synchronization word parameters. If unset, the default synchronization word defined
+  // by the AX100 will be used.
+  SynchronizationParams synchronization_params = 1;
+
+  // Whether or not CCSDS scrambling is used
+  bool ccsds_scramble_enabled = 4;
```

Comment:
For boolean flags like these, I find there are two naming patterns in this code base. One is `is_ccsds_scramble_enabled` in monitoring.proto and the other is `enable_ccsds_scramble` in stellarstation.proto. It's a bummer that this small API already has inconsistent naming  but let's not add the third pattern. `is_ccsds_scrable_enabled` would be preferred because this pattern is used much more than the other.

Reply:
While there might be misuses, I think in general it's normal for there to be two patterns, one for a proto describing state and one describing parameters. Since this is a parameter, verb seems fine?

Reply:
Makes sense!